lib/resources/r4/immunization.yaml

```diff
@@ -15,6 +15,31 @@ fields:
     }
   note: resourceType must be Immunization
 
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  url: http://hl7.org/fhir/r4/json.html#resources
+  description: The type of the FHIR resource.
+  action: update
+  example: |
+    {
+      "resourceType": "Immunization"
+    }
+  note: resourceType must be Immunization
+
+- name: id
+  required: 'Yes'
+  cardinality: 0..1
```

Comment:
id should have cardinality 1..1 indicating it's required

Reply:
Updated it. https://github.com/cerner/fhir.cerner.com/pull/295/commits/fe9f32d2c48645c5ba462f17793818a3e32b5d4a#diff-2c50e8f41daaba3a640861e7b6e81488R22